app/components/concept-page/content.ts

```diff
@@ -28,16 +38,6 @@ export default class ContentComponent extends Component<Signature> {
     element.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
   }
 
-  @action
-  handleConceptDidUpdate() {
-    this.currentProgressPercentage = 0;
-  }
```

Comment:
can you confirm if removing this works? i feel like we don't want to reset the progress to 0 on concept update

Reply:
@libmartinito this was intended to handle cases where you navigate from one concept to another.

If you go to a concept that is part of a collection and there's a next concept in the collection, here's what should happen:

https://github.com/codecrafters-io/frontend/assets/3893573/213c56e9-283c-43d1-966c-b2f859c86305

Without this hook, previously, the page would've update to the new concept but the progress would remain the same (100%). 

Should be safe to remove now since `conceptEngagement` is fetched in the route and passed down - so when `Concept` changes, `conceptEngagement` will change too.